feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/login/SignupActivity.java

```diff
@@ -36,41 +42,62 @@ public void cancelSignup(View view) {
     }
 
     public void confirmSignup(View view) {
-        // THIS DOES NOT CHECK FOR UNIQUE USERNAMES YET. SHOULD CHECKING GO INTO SERVICE?
-
```

Comment:
when I tried to add another user called rehab (which already exists), it did not let me. So it seems it does check for unique usernames? 
Im assuming this comment is out of date.

Reply:
ignore me, did not realize this was red oops.